package.json

```diff
@@ -20,9 +20,18 @@
   },
   "homepage": "https://github.com/NatanimA/To-Do-List-App#readme",
   "devDependencies": {
```

Comment:
- [ ] I should be able to run your project with the start command, kindly check your scripts and webpack 
![image](https://user-images.githubusercontent.com/62705947/192562631-b86fabba-d2c9-4c8c-8672-9de48ec0dc52.png)

Reply:
It is because I added node_modules in gitignore, should I remove it and push that as well? @GzimAsani

Reply:
No, node_modules should be in gitignore

Reply:
Then why is it not working?, It is working on mine? @GzimAsani ,Please any suggestions

Reply:
> No, node_modules should be in gitignore

> It is working on mine you see. @GzimAsani 

![Capture](https://user-images.githubusercontent.com/66979348/192569027-b78f023c-4230-47d3-96c7-0dc63452da9f.JPG)